roseus/euslisp/roseus-utils.l

```diff
@@ -1137,7 +1140,6 @@
         (unix::usleep (* 50 1000))
         (ros::spin-once)
         (if lmsg (return))))
-    (if unsubscribe (ros::unsubscribe topic-name))
```

Comment:
It seems like you are forgetting to unsubscribe.

Reply:
Thank you.
I add (ros::unsubscribe)